uktrade/data-workspace-frontend
# Validation for the Descriptive name page

Next step of add table journey: [the validation for the Descriptive name page.](https://uktrade.atlassian.net/jira/software/projects/DT/boards/356?selectedIssue=DT-2056)

**How to Test** 
1. Head to /datasets
2. Add /add-table to the end of the dataset ID
3. Click through the flow to get to 'Give your table a descriptive name'
4. Try 'data', 'dataset', 'record', '_' and an empty string in the input field to confirm the error messages are displaying correctly. 
5. Enter a valid string to proceed, currently being redirected to '/' to be changed once the nest step in the flow is completed.
